File: articles/documentation-model-cards-and-datasheets-for-algorithms/typescript/documentation_quality_score.ts
const scores = [0.62, 0.6875, 0.58, 0.50, 0.56, 0.52];
const quality = scores.reduce((a, b) => a + b, 0) / scores.length;
console.log(`documentation_quality_score=${quality.toFixed(4)}`);

File: articles/documentation-model-cards-and-datasheets-for-algorithms/haskell/DocumentationQualityScore.hs
module Main where

main :: IO ()
main = do
  let scores = [0.62, 0.6875, 0.58, 0.50, 0.56, 0.52]
      quality = sum scores / fromIntegral (length scores)
  putStrLn ("documentation_quality_score=" ++ show quality)
